en/frames_vtol/tailsitter.md

```diff
@@ -0,0 +1,151 @@
+The tailsitter rotors are permanently fixed in position for forward flight.
+
+<img src="../../assets/airframes/vtol/wingtraone/hero.jpg" title="Wingtra: WingtraOne VTOL Duo Tailsitter" alt="wingtraone" />
+
+Tailsitters [have the same benefits as other VTOL vehicles](../frames_vtol/README.md) vehicles: vertical takeoff and landing, fast and efficient fixed wing flight, and hovering.
+Compared to other types of VTOL, tailsitters are simple and robust, but can be harder to control (particularly in wind).
+They tradeoff efficiency in hover and forward flight because they use the same rotors for both.
+
+This topic topic describes the different tailsitter configurations, and provides information required for assembly and PX4 setup.
+
+## Tailsitter Types
+
+PX4 supports two VTOL tailsitter configurations.
+The [VTOL Duo Tailsitter](../airframes/airframe_reference.md#vtol-duo-tailsitter) has two rotors and uses elevons to flip from hover into fixed-wing flight.
+The [VTOL Quad Tailsitter](../airframes/airframe_reference.md#vtol-quad-tailsitter) has four rotors like a quadcopter, and uses them (along with optional elevons, if present) to transition between modes.
+
+<div class="grid_wrapper two_column">
+  <div class="grid_item">
+    <div class="grid_item_heading"><big>Duo Tailsitter</big></div>
+    <div class="grid_text">
+    Two rotors and elevons.
+    <img src="../../assets/airframes/vtol/wingtraone/hero.jpg" title="Wingtra: WingtraOne VTOL Duo Tailsitter" alt="wingtraone" /> 
+    <br><br>
+    <ul>
+      <li>More efficient in forward flight</li>
+      <li>Harder to control in hover, particularly in wind</li>
+      <li>Harder to tune/configure</li>
+    </ul>
+    </div>
+  </div>
+<div class="grid_item">
+  <div class="grid_item_heading"><big>Quad Tailsitter</big></div>
+  Four rotors and optional elevons.
+  <div class="grid_text">
+  <img title="Skypull SP-1 VTOL QuadTailsitter" src="../../assets/airframes/vtol/skypull/skypull_sp1.jpg" />
+  <ul>
+    <li>Easier and more stable hover flight</li>
+    <li>Less compact form factor</li>
+  </ul>
+  </div>
+</div>
+</div>
+
+Quad Tailsitters are easier to fly in hover mode, and more stable - particularly in windy conditions.
+Duo Tailsitters are generally more efficient in cruise flight (4 smaller propellers are less efficient than 2 larger ones) and physically more compact.
+
+From a controls/aerodynamics perspective the Duo Tailsitter is much more complex in hover mode.
+As a result they are much harder to tune for hover and transitioning. 
+
+
+## Flying and Flight Modes
+
+The flight modes for VTOL vehicles are the same as for [multicopter](../getting_started/flight_modes.md#mc_flight_modes) when flying in MC mode and [fixed-wing](../getting_started/flight_modes.md#fw_flight_modes) when flying in FW mode.
+
+For more information see [VTOL > Flying and Flight Modes](../frames_vtol/README.md#flying-and-flight-modes)
+
+
+## Assembly
```

Comment:
@sfuhrer Thanks for having a look at this. That's the kind of feedback I need.

What I'm trying to do in the sub pages is help people understand what these frames really look like and explain the pros and cons of each. In particular, there are cases where it is pretty hard to work out what the vehicle IS if you don't have a proper image. Case in point:
![image](https://user-images.githubusercontent.com/5368500/129181249-8e534aab-a801-47f8-bf68-9c36bb820223.png)

So that's the bit I like about this so far. I have been trying to make the pages kind of a mini entry point, so that you don't have to jump too much to find all the relevant info. On the other hand, I don't want heaps of duplication all over the place, as that means maintenance is harder.

Upshot, yes, I think I do need to move some of this up a level, but I'm not quite sure what yet.

Reply:
alright, that makes perfectly sense :+1: